packages/perseus-editor/src/components/start-coord-settings.tsx

```diff
@@ -67,11 +112,43 @@ const StartCoordSettings = (props: Props) => {
     const {type, range, step, onChange} = props;
     const [isOpen, setIsOpen] = React.useState(true);
 
-    if (type !== "linear" && type !== "ray") {
+    if (
+        type !== "linear" &&
+        type !== "ray" &&
+        type !== "segment" &&
+        type !== "linear-system"
+    ) {
```

Comment:
I'm going to remove these checks in the next PR since this is behind a feature flag anyway.